lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -8392,23 +8392,37 @@ bool SwiftASTContext::CacheUserImports(SwiftASTContext &swift_ast_context,
   return true;
 }
 
+namespace {
+struct CUSignature : public std::pair<Module *, lldb::user_id_t> {
+  CUSignature(CompileUnit &compile_unit)
```

Comment:
What are the advantages to a wrapper type vs a function such as `std::pair<Module *, lldb::user_id_t> CUSignature(CompileUnit &)`?

Reply:
Originally I thought I'd keep a local variable with the key into the map around, but it didn't.